model/event_test.go

```diff
@@ -49,7 +49,7 @@ func TestEventRefUnmarshalJSON(t *testing.T) {
 			desp:   "invalid json format",
 			data:   `{"invoke": 1}`,
 			expect: EventRef{},
-			err:    `json: cannot unmarshal number into Go struct field eventRefForUnmarshal.invoke of type model.InvokeKind`,
+			err:    `eventRef.invoke must be an sync,async`,
```

Comment:
```suggestion
			err:    `eventRef.invoke must be sync or async`,
```
can this message be updated to the example above?